db/models/getArticles.model.js

```diff
@@ -1,11 +1,45 @@
 const db = require("../connection");
 
-exports.findArticles = () => {
-  return db
-    .query(
-      `SELECT 
+exports.findArticles = (topic) => {
+  if (topic) {
```

Comment:
OPTIONAL(but recommended)  - How could we account for both situations (topic present or not) with a single SQL query and add an extra line of SQL if a topic is present?